### DISCUSSION TITLE
Packages to use BLE

### DISCUSSION THREAD

**Discussion by Anxo42** [kieranabrennan/every-breath-you-take#10] [Q&A] [answered]
Hello! I'm working on my degree final project and it's similar to your application. My questions is: which python packages did you use to can communicate your aplication by BLE?
Best regards!

**Comment by kieranabrennan** [comment, marked as answer]
@Anxo42 the package used to communicate to a BLE device using Python is Bleak
https://bleak.readthedocs.io/en/latest/
